### ISSUE TITLE
[BUG] Can't download dxvk on first run

### ISSUE THREAD

**Issue by teratux** [bottlesdevs/Bottles#836]
**Describe the bug**
When running the installer the runners download fails with a continous error message:
```
(12:41:02) INFO Installing component: [dxvk-1.9.2]. 
(12:41:03) ERROR Extraction failed! Archive ends earlier than expected. 
(12:41:03) WARNING No dxvk found.
```

**Installation**
 - Method: AppImage
 - Version 2021.12.14-treviso-3

**To Reproduce**
1. Download installer
2. Run installer
3. Click on Next when asked to download ~70MB of runners

**Expected behavior**
The runners should download

**Desktop (please complete the following information):**
 - OS: openSUSE
 - Version 15.1
 - Display server: Xorg

**Comment by mirkobrombin** [comment]
It appear when there is no stable connection, I'm already aware about this. Are you able to install from Preferences -> DXVK ?

**Comment by teratux** [comment]
> It appear when there is no stable connection, I'm already aware about this. Are you able to install from Preferences -> DXVK ?

none of the buttons I highlighted are clickable and the dialog can only be navigated up to the Install option
![image](https://user-images.githubusercontent.com/92475/146846898-29008909-16c3-417c-8f02-2ce56d2cd699.png)

**Comment by mirkobrombin** [comment]
Bottles UI is completely broken in this screenshot, also icons are wrong. Btw follow the onboard one more time, if it doesn’t work I suggest to use the flatpak

**Comment by Bananaman** [comment]
@teratux I am like you. I used openSUSE and now Fedora because I want fresh native packages.

But in this case, this is actually an app that runs much better as a Flatpak. The Flatpak ensures thst all components are compatible with each other. It is more tested. It doesn't have OS specific bugs.

And above all, the Flatpak version protects your Host OS against Windows malware and home folder clutter.

I have an ongoing thread where I am learning about the Flatpak. Check it out:

https://github.com/bottlesdevs/Bottles/issues/838

**Comment by mirkobrombin** [comment]
should be fixed by 29567a14e2485650fce3df3ef665b96cb7b1d77a will be distributed with next release

**Comment by Bananaman** [comment]
@mirkobrombin Oh wow, very cool! :heart:
